library.properties

```diff
@@ -1,5 +1,5 @@
 name=STM32duino RTC
-version=1.0.3
+version=1.0.4
```

Comment:
Please remove this change. This is up to the maintainers to define the next version.